extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,257 @@
+            "undefined": "Not defined",
+            "unknown_user":
+              "The address does not exist. There is no use sending deliveries to this address.",
+            "invalid_domain":
+              "The domain of the email address is incorrect or no longer exists.",
+            "unreachable":
+              "An error has occurred in the message delivery chain.These addresses can be removed from the quarantine list to make another attempt.",
+            "disabled":
+              "The user uses a messaging service which is accessible from the web. When the Internet Access Provider (IAP) detects a lengthy period of inactivity, it can close the user's account. Deliveries to the user's address will then be impossible.",
+            "mailbox_full":
+              "The recipient's mailbox contains too many messages. This address can be removed from the quarantine list to make another attempt.",
+            "not_connected":
+              "The recipient's mobile phone is switched off or not connected to the network when the message is sent.",
+            "refused":
+              "The address was rejected following the application of a security rule (e.g. by an anti-spam program)",
+            "error_ignored":
+              "The address is whitelisted and the message ignored.",
+            "address_undefined": "No address is given for the recipient.",
+            "blacklisted":
+              "The address was blacklisted at the time of sending.",
+            "quarantine":
+              "The address was in quarantine at the time of sending.",
+            "duplicate":
+              "The address of the recipient was already in this campaign activity.",
+            "typology_rule":
+              "The recipient was excluded by a 'SQL' type campaign typology rule.",
+            "business_ranking":
+              "The recipient was excluded by an 'arbitration' type campaign typology rule",
+            "cancelled": "Delivery cancelled",
+            "quality": "The quality rating for this address is too low",
+            "unchecked": "Unqualified address",
+            "offer_missing": "Not eligible for the offers",
+            "over_delivery_limit":
+              "The campaign activity had more than allowed number of target profiles.",
+            "expired":
+              "The campaign activity exceeded the time duration to complete.",
+            "too_long": "Text too long",
+            "untranslatable": "Character not supported by encoding",
+            "control_group": "Control address"
+          },
+          "description":
+            "The reason why the message could/would not be delivered.",
+          "default": "Undefined"
+        },
+        "https://ns.adobe.com/experience/campaign/message/reasonMessage": {
+          "title": "Reason Message",
+          "type": "string",
+          "description":
+            "Reason of message delivery failure when the status is `bounce` or `nonDeliverable`."
```

Comment:
Ok, I've been searching for a property `status` and could not find it. Same for the values `bounce` and `nonDeliverable`. The only place where they are mentioned is here. You are either missing a property, or the description is off.

Reply:
Fixing it.